languages/csharp/CommandRunner.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Bitwarden.Sdk;
+using System.Text.Json;
+
+internal class CommandRunner
+{
+
+    private readonly BitwardenSafeHandle _handle;
+
+    internal CommandRunner(BitwardenSafeHandle handle)
+    {
+        _handle = handle;
+    }
+
+    internal T? RunCommand<T>(Command command)
+    {
+        var req = command.ToJson();
+        var result = BitwardenLibrary.RunCommand(req, _handle);
+        return JsonSerializer.Deserialize<T>(result);
```

Comment:
We should use the `JsonSerializerOptions` generated in the `schemas.cs` to ensure enums and dates are serialized correctly.

```suggestion
        return JsonSerializer.Deserialize<T>(result, Converter.Settings);
```

Reply:
changed